### DISCUSSION TITLE
PowerShell Scripts inventory?

### DISCUSSION THREAD

**Discussion by jengstrom440** [semaphoreui/semaphore#2230] [General]
Question regarding the PowerShell scripts, is there going to be an inventory to communicate to the servers with an inventory file? I am assuming that we don't want to add the hosts into the actual code itself? I wanted to just run a $psversiontable, assuming I would get the output in a json format, but as I was making one, I did not see a way to direct that to a server to run, like a regular ansible job to run of winrm 5986. 

Also, is possible to have a dynamic inventory built in so one can build, let's say a dynamic inventory of windows servers from active directory? That would be excellent! Thanks!

**Comment by fiftin** [comment]
Hi @jengstrom440 thank you, I created an issue for your idea.
